src/EntryList.js

```diff
@@ -0,0 +1,12 @@
+function EntryList(props) {
+
+    return
+    (
+        <ul>
+            let fruits =    {fruits.map(function (fruit) {
```

Comment:
Please remove `let fruits =    ` and use the props parameter instead 
`props.fruits`